include/lsst/meas/base/CentroidUtilities.h

```diff
@@ -44,21 +44,21 @@ struct CentroidResult {
     CentroidResult();
 
     /// Constructor; initializes everything from values.
-    explicit CentroidResult(CentroidElement _x, CentroidElement _y, CentroidCov const & matrix):
-        x(_x),
-        y(_y),
+    explicit CentroidResult(CentroidElement x, CentroidElement y, CentroidCov const & matrix):
+        x(x),
+        y(y),
```

Comment:
I don't think we want to change this, or if we do, we want to change the arguments to e.g. `x_` rather than `x`.  The Python arguments can still be `x`, `y`, of course, but I think it's best to avoid any confusing about which `x` and `y` are in play here.